effekt/shared/src/main/scala/effekt/generator/llvm/Transformer.scala

```diff
@@ -428,7 +428,7 @@ object Transformer {
       case machine.Type.Prompt()     => 8 // TODO Make fat?
       case machine.Type.Stack()      => 8 // TODO Make fat?
       case machine.Type.Int()        => 8 // TODO Make fat?
-      case machine.Type.Byte()       => 8
+      case machine.Type.Byte()       => 1
```

Comment:
Didn't @phischu just change this from 1 to 8 to fix something else?

Reply:
This was an insufficient fix as described in https://github.com/effekt-lang/effekt/pull/785#issuecomment-2607553642